### PR TITLE
Added tasks for homework lesson 16

### PR DESCRIPTION
**Pull request by max3koz**
Added tasks for homework lesson 16

### CODE REVIEW

File: lesson_l16/task_01/developer.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from lesson_l16.task_01.employee import Employee
+
+
+class Developer(Employee):
+    programming_language: str
```
Comment (DartBrovsky) [inline lesson_l16/task_01/developer.py]: such attributes is better to be private, and use getter methods to provide access for that.

File: lesson_l16/task_01/logging_config.ini [hunk 2]
```diff
@@ -0,0 +1,34 @@
+[loggers]
```
Comment (DartBrovsky) [inline lesson_l16/task_01/logging_config.ini:1]: You can create once your logger_ini file for whole project in root. Then in all your modules you can unly reuse it.

**Review by DartBrovsky — APPROVED** [review]
I approve this and leave some comments. Please take a look into them. Thanks.

File: lesson_l16/task_01/logging_config.ini (hunk 2 above)
Comment (max3koz) [reply lesson_l16/task_01/logging_config.ini:1]: In the next task, I`ll refactor the code and move the logger init file to the root of the AQA course.

File: lesson_l16/task_01/developer.py (hunk 1 above)
Comment (max3koz) [reply lesson_l16/task_01/developer.py]: Updated in commit 32fb69f57063920828ca5fa16dcb7698619bada9

**Review by DartBrovsky — APPROVED** [review]
approved

**Pull request merged**